src/main/java/com/github/complate/nashorn/renderer/NashornComplateRenderer.java

```diff
@@ -49,8 +76,9 @@ private NashornComplateRenderer(ComplateScript script, Map<String, ?> bindings)
         }
 
         engine.put("javax.script.filename", script.getDescription());
-        try (Reader reader = readerFor(script)) {
+        try (final Reader reader = readerFor(script)) {
```

Comment:
I decided not to use `final` in these cases. To keep the overall look of the code "identical" I would like to see the line like:

```suggestion
        try (Reader reader = readerFor(script)) {
```